src/components/SRNEntry.vue

```diff
@@ -130,13 +133,30 @@ export default {
         this.validateCount,
         this.isSingleEntryOnly,
         this.redirectID,
-        this.doesUserExist
+        this.isUserValid,
+        this.purpose,
+        this.purposeParams,
+        this.redirectTo
       );
 
       userIsValidated.then((result) => {
```

Comment:
Instead of using `then` can you try to use just async await here? That is much cleaner than `then` chaining. No hard and fast rule, just looks cleaner. If you have time then give it a try, will be good practice for you!

Reply:
Will do.

Reply:
Will do.